Core/PSPLoaders.cpp

```diff
@@ -355,3 +356,11 @@ bool Load_PSP_ELF_PBP(FileLoader *fileLoader, std::string *error_string) {
 
 	return __KernelLoadExec(finalName.c_str(), 0, error_string);
 }
+
+bool Load_PSP_GE_Dump(FileLoader *fileLoader, std::string *error_string) {
+	BlobFileSystem *umd = new BlobFileSystem(&pspFileSystem, fileLoader, "data.ppdmp");
+	pspFileSystem.Mount("disc0:", umd);
```

Comment:
Hm, I guess that making it easy to run on mobile might make it worthwhile to have an extra FileSystem.